python/tests/data/svg/tree.svg

```diff
@@ -1,47 +1 @@
-          <text class="rgt">0</text>
-        </g>
-      </g>
-      <g class="node n5 p9" transform="translate(36.0 138.294)">
-        <g class="leaf node n2 p5 sample" transform="translate(-18.0 1.70594)">
-          <path class="edge" d="M 0 0 V -1.70594 H 18.0"/>
-          <circle cx="0" cy="0" r="1"/>
-          <text>2</text>
-        </g>
-        <g class="leaf node n3 p5 sample" transform="translate(18.0 1.70594)">
-          <path class="edge" d="M 0 0 V -1.70594 H -18.0"/>
-          <circle cx="0" cy="0" r="1"/>
-          <text>3</text>
-        </g>
-        <path class="edge" d="M 0 0 V -138.294 H -36.0"/>
-        <circle cx="0" cy="0" r="1"/>
-        <text class="lft">5</text>
-      </g>
-      <circle cx="0" cy="0" r="1"/>
-      <text>9</text>
-    </g>
-  </g>
-</svg>
+<svg baseProfile="full" height="200" id="XYZ" version="1.1" width="200" xmlns="http://www.w3.org/2000/svg" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><style type="text/css"><![CDATA[.axis {font-weight: bold}.tree, .axis {font-size: 14px; text-anchor:middle}.edge {stroke: black; fill: none}.node > .sym {fill: black; stroke: none}.tree .lab {dominant-baseline: middle}.tree .lab.lft {text-anchor: end}.tree .lab.rgt {text-anchor: start}.mut {fill: red; font-style: italic}]]></style><style type="text/css"><![CDATA[.edge {stroke: blue}]]></style></defs><g class="tree t1"><g class="node n7 p0 root" transform="translate(100.0 30.0)"><g class="a7 node n4 p0" transform="translate(-36.0 138.519)"><g class="a4 leaf node n0 p0 sample" transform="translate(-18.0 1.4814)"><path class="edge" d="M 0 0 V -1.4814 H 18.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab" transform="translate(0 12)">0</text></g><g class="a4 leaf node n1 p0 sample" transform="translate(18.0 1.4814)"><path class="edge" d="M 0 0 V -1.4814 H -18.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab" transform="translate(0 12)">1</text></g><path class="edge" d="M 0 0 V -138.519 H 36.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab lft" transform="translate(-3 -6)">4</text></g><g class="a7 node n5 p0" transform="translate(36.0 133.294)"><g class="a5 leaf node n2 p0 sample" transform="translate(-18.0 6.70591)"><path class="edge" d="M 0 0 V -6.70591 H 18.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab" transform="translate(0 12)">2</text></g><g class="a5 leaf node n3 p0 sample" transform="translate(18.0 6.70591)"><path class="edge" d="M 0 0 V -6.70591 H -18.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab" transform="translate(0 12)">3</text></g><path class="edge" d="M 0 0 V -133.294 H -36.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab rgt" transform="translate(3 -6)">5</text></g><circle class="sym" cx="0" cy="0" r="3" /><text class="lab" transform="translate(0 -10)">7</text></g></g></svg>
```

Comment:
Can we run these through `tidy` or similar?

Reply:
Jerome mentioned the "debug" option, which I guess we should use in this case (and I'm not sure why it's not working)